### ISSUE TITLE
Importing object of css`` keys results in displayName error

### ISSUE THREAD

**Issue by seaneking** [callstack/linaria#683]
## Environment

- Linaria version: 1.3.3
- Bundler (+ version): Webpack
- Node.js version: 12.6
- OS: macOS Big Sur

Other env:
- Storybook v6
- Svelte
- Typescript

## Description

When using Linaria in a Storybook + Svelte environment, using the `css` function in my Svelte components works perfectly, but importing styles defined in another typescript file (eg: for shared styles) results in the following hard to debug error:

```
TypeError: {...}/typesets.ts: displayName.charAt is not a function
 at TaggedTemplateExpression ({...}/node_modules/linaria/src/babel/visitors/TaggedTemplateExpression.js:133:20)
    at TaggedTemplateExpression ({...}/node_modules/linaria/src/babel/extract.js:28:15)
    at NodePath._call ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/path/context.js:92:31)
    at TraversalContext.visitQueue ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:116:16)
    at TraversalContext.visitSingle ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:88:19)
    at TraversalContext.visit ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:144:19)
    at Function.traverse.node ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/index.js:82:17)
    at NodePath.visit ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/path/context.js:99:18)
    at TraversalContext.visitQueue ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:116:16)
    at TraversalContext.visitMultiple ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:83:17)
    at TraversalContext.visit ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:142:19)
    at Function.traverse.node ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/index.js:82:17)
    at NodePath.visit ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/path/context.js:99:18)
    at TraversalContext.visitQueue ({...}/node_modules/linaria/node_modules/@babel/traverse/lib/context.js:116:16)
```

## Reproducible Demo

My webpack config is pretty standard, just adding Linaria's loader to Storybook's default babel rule:

```js
config.module.rules[0].use.push({
  loader: 'linaria/loader',
  options: {
    sourceMap: !prod
  }
});
```

No idea how I can debug this. I tried setting `displayName` to true and fiddling with my config, but to no avail. This is the const I'm trying to import:

```js
import { css } from 'linaria';

export const text = {
  16: css`
    font-size: 1rem;
  `,
  18: css`
    font-size: 1.125rem;
    font-weight: 500;
    line-height: 1;
  `,
  20: css`
    font-size: 1.25rem;
  `,
  24: css`
    font-size: 1.5rem;
  `,
  42: css`
    font-size: 2.625rem;
    line-height: 1.15;
  `,
  50: css`
    font-size: 3.125rem;
    line-height: 1.2;
  `
};
```

**Comment by seaneking** [comment]
Okay figured it out, if I flatten that object and export `css` consts directly it works. Is this limitation documented anywhere?

I'll update the issue title to reflect.

**Comment by Anber** [comment]
Hi @seaneking 
Thank you for your report.
The problem here in numeric keys in `text` objects. Linaria tries to use the first letter of a key as the first letter of a class name, but it fails because there is no method `charAt` for numbers. This bug was accidentally fixed in 1.4/2.0, but there is still a problem – in case of numeric keys, it generates invalid class names that start with a number. The fix will be included in the next 1.4/2.0 RC.
